src/components/Modals/Send/hooks/useFormSend/useFormSend.test.tsx

```diff
@@ -168,6 +168,16 @@ describe.each([
         ]),
     )
 
+    useSelectorMock.mockReturnValue({
+      [formData[SendFormFields.AccountId]]: {
+        bip44Params: {
+          purpose: 44,
+          coinType: 60,
+          accountNumber: 0,
+        },
+      },
+    })
```

Comment:
It seems like we mock the same data in every test. Can we de-dupe by adding to our `beforeEach`?